language-support/ts/package.json

```
@@ -0,0 +1,12 @@
+{
+  "dependencies": {
+    "typescript": "3.1.6",
+    "@mojotech/json-type-validation": "^3.1.0",
+    "cross-fetch": "^3.0.4"
+  },
+  "devDependencies": {
+    "@bazel/bazel": "0.29.0",
```

```suggestion
    "@bazel/bazel": "1.1.0",
```
Shouldn't this track the Bazel version? At least the [released versions](https://www.npmjs.com/package/@bazel/bazel?activeTab=versions) seem to track Bazel versions.

---

changed it to 1.1.0